k8s/bases/swe-btcc/swe-btcc-data-bridge-configmap.yaml

```diff
@@ -0,0 +1,133 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: swe-btcc-data-bridge
+  labels:
+    app: gridcapa
+    component: data-bridge
+    region: swe
+    process: btcc
+data:
+  application.yml: |-
+    data-bridge:
+      configuration:
+        zone-id: "UTC"
+        target-process: SWE_BTCC
+        files:
+          - file-type: BOUNDARY_EQ
+            file-regex: (?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})T(?<hour>[0-9]{2})(?<minute>[0-9]{2})Z__ENTSOE_EQBD_[0-9]{3}.(xml|XML)
+            time-validity: yearly
+            remote-file-regex: ".*.zip|(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})T(?<hour>[0-9]{2})(?<minute>[0-9]{2})Z__ENTSOE_EQBD_[0-9]{3}.(xml|XML)"
+            source-directory: "/boundaries"
+            sink-directory: "/BOUNDARIES"
+            file-list-persistence-file: /data/gridcapa/swe-btcc/ftp-metadata-store.boundary-eq.properties
+          - file-type: BOUNDARY_TP
+            file-regex: (?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})T(?<hour>[0-9]{2})(?<minute>[0-9]{2})Z__ENTSOE_TPBD_[0-9]{3}.(xml|XML)
+            time-validity: yearly
+            remote-file-regex: ".*.zip|(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})T(?<hour>[0-9]{2})(?<minute>[0-9]{2})Z__ENTSOE_TPBD_[0-9]{3}.(xml|XML)"
```

Comment:
What is the behaviour if you drop a `*.zip` file in `/boundaries` folder, since both BOUNDARY_EQ and BOUNDARY_TP have it in their `remote-file-regex`?

Reply:
Good question, by default for a few months now, the files should be zipped and have a "new" specific name. With databridge V2 it is the unzipped fie-regex ( and not the remote-file-regex) that is used to determine the file type :)